lfucg-exactions/server/server/settings/base.py

```diff
@@ -234,7 +234,7 @@ def custom_show_toolbar(request):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'exactions.db',
+        'NAME': 'exactions',
```

Comment:
Was this recommended by Chris or Wei-Keat?  Just curious.  Also just need to make sure this file is gitignored